packages/engine/src/physics/systems/PhysicsSystem.ts

```diff
@@ -134,8 +156,13 @@ export default async function PhysicsSystem(world: World) {
 
     // step physics world
     const substeps = engineState.physicsSubsteps.value
-    world.physicsWorld.timestep = engineState.fixedDeltaSeconds.value / substeps
+    const timestep = engineState.fixedDeltaSeconds.value / substeps
+    world.physicsWorld.timestep = timestep
+    const smoothAlpha = 50 * timestep
```

Comment:
Why 50? What does this mean?

Reply:
It's a smoothness multiplier

Reply:
let's try and avoid magic numbers and give this a name